docus/content/rules/0.index.md

```diff
@@ -25,16 +26,29 @@ We have documented our [moderation process](/rules/process) to be as open as pos
 
 ### Defederation
 
-We actively "defederate" vmst·io from other instances that are incompatible with our fundamental rules, in an effort to protect our users.
-The decision to defederate can come from reports by our users but is also based on community discussion around such bad actors.
+We "defederate" vmst·io from other instances that are incompatible with our fundamental rules, in an effort to protect our users and limit our liability.
+The decision to defederate can come from reports by our users but is also based on community discussion around such bad actors, including the perodic import of community maintained blocklists.
 We would rather be proactive in blocking bad actors versus waiting for abuse to happen.
 
-We previously listed all defederated instances on at [vmst.io/about](https://vmst.io/about), but have since switched to posting the listing in CSV format on our GitHub.
+We list all defederated instances at [vmst.io/about](https://vmst.io/about), but also periodically post the full listing in CSV format on our GitHub.
 
 :button-link[Download Blocklist]{icon="mdi:file-download-outline" href="https://github.com/vmstan/vmstio/blob/main/domain_blocks.csv" blank}
 
-The rationale for this change was to limit the listing of domains that sometimes contained offensive language or hosted illegal material.
-Being posted in CSV also allows other instance administrators to easily import our blocklist or compare it against their own.
+The rationale for this is our regular listing includes domains that sometime contain offensive language or hosted illegal material, and are marked as sensitive.
+Being posted in CSV allows other instance administrators to easily import our blocklist or compare it against their own.
+
+#### Blocklist Imports
+
+We believe community maintained blocklists serve an important purpose to help jumpstart the protection of users.
+
+The foundation of our defederation list is the Oliphant Blocklist at Tier 0.
+We also incorporate the Tier 1 blocklist after careful review and application of our local whitelist.
+
+Concensus based lists should have a high number of members to prevent the decisions of one or two administrators from imposing their will on the rest of the Fediverse.
```

Comment:
```suggestion
Consensus based lists should have a high number of members to prevent the decisions of one or two administrators from imposing their will on the rest of the Fediverse.
```